runtime/drivers/repo.go

```diff
@@ -23,3 +24,5 @@ type RepoStore interface {
 type RepoObjectStat struct {
 	LastUpdated time.Time
 }
+
+var FileAlreadyExists = errors.New("file already exists")
```

Comment:
Go convention is prefixing const errors with `Err`. In this case, consider just using `os.ErrExist`